src/openrct2/object/ObjectManager.cpp

```diff
@@ -396,16 +398,15 @@ class ObjectManager final : public IObjectManager
         Guard::ArgumentNotNull(object, GUARD_LINE);
 
         auto result = std::numeric_limits<size_t>().max();
-        auto it = std::find_if(
-            _loadedObjects.begin(), _loadedObjects.end(), [object](auto& obj) { return obj.get() == object; });
+        auto it = std::find_if(_loadedObjects.begin(), _loadedObjects.end(), [object](auto& obj) { return obj == object; });
```

Comment:
I think this can be replaced by a simple `std::find` call, since all it does it compare pointers.

Reply:
True